src/socket.rs

```diff
@@ -534,6 +534,22 @@ impl UtpSocket {
         }
     }
 
+    #[cfg(windows)]
+    fn ignore_udp_error(e: &Error) -> bool {
+        match e.raw_os_error() {
+            Some(e) => match e {
+                10054 | 10040 => true,
```

Comment:
The commit message elaborates a lot on this behaviour. I originally shared [my finding on the issue at the #454 PR](https://github.com/maidsafe/crust/pull/454#issuecomment-167689738).

Do you want me to add a comment before the code or something like that? If you find the explanation of the commit message confusing, can you elaborate which part was confusing?